docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -17,7 +17,7 @@ Generating unit files for a pod requires the pod to be created with an infra con
 - Note: The generated `podman run` command contains an `--sdnotify` option with the value taken from the container.
 If the container does not have any explicitly set value or the value is set to __ignore__, the value __conmon__ is used.
 The reason for overriding the default value __container__ is that almost no container workloads send notify messages.
-Systemd would wait for a ready message that never comes, if the value __container__ is used for a container
+Systemd waits for a ready message that never comes, if the value __container__ is used for a container
```

Comment:
I think "will wait" is better than "waits" in this context, but it is a coinflip.

```suggestion
Systemd will wait for a ready message that never comes if the value __container__ is used for a container
```